lib/rubocop/cop/mixin/multiline_literal_brace_layout.rb

```diff
@@ -46,6 +36,30 @@ def autocorrect(node)
 
       private
 
+      def handle_new_line(node)
+        return unless closing_brace_on_same_line?(node)
+
+        add_offense(node, :expression, self.class::ALWAYS_NEW_LINE_MESSAGE)
+      end
+
+      def handle_same_line(node)
```

Comment:
Only this code was added. The rest of the code has been moved from `check_brace_layout` for clarity.